samples/word/20-lists/manage-formatting.yaml

```diff
@@ -0,0 +1,118 @@
+order: 2
+id: word-lists-manage-formatting
+name: List formatting
```

Comment:
Since you're creating a list, can this sample just be called "Lists"?

Reply:
There's already another sample that shows how to insert a list